<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui</class>
 <widget class="QWidget" name="ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>496</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>496</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>744</width>
    <height>496</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CWNU选课GUI_Demo</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>721</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>配置区</string>
   </property>
   <widget class="QComboBox" name="select_type">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>20</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选课类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>所有课程：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="all_course">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>60</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>监控间隔：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="yc_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>单位: ms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>721</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>运行日志</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTextBrowser" name="log_plain"/>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="cookie_plain">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>11</y>
     <width>560</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>11</y>
     <width>56</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>cookie:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>11</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
